src/main/scala/com/advancedtelematic/campaigner/db/Repository.scala

```diff
@@ -127,74 +123,24 @@ protected [db] class DeviceUpdateRepository()(implicit db: Database, ec: Executi
         case _ => DBIO.failed(DeviceNotScheduled)
       }.map(_ => ())
 
-  def persistMany(updates: Seq[DeviceUpdate]): Future[Unit] = db.run {
-    DBIO.sequence(updates.map(Schema.deviceUpdates.insertOrUpdate)).transactionally.map(_ => ())
-  }
-}
-
-protected [db] class GroupStatsRepository()(implicit db: Database, ec: ExecutionContext) {
```

Comment:
We lose group stats, but that feature is one on the roadmap, including showing failure codes by group. Okay for now.